sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/EndToEndTimeOutValidationTests.java

```diff
@@ -248,7 +248,7 @@ public void clientLevelEndToEndTimeoutPolicyInOptionsShouldTimeout() {
             // with injected delay
             CosmosQueryRequestOptions queryRequestOptions = new CosmosQueryRequestOptions()
                 .setCosmosE2EOperationRetryPolicyConfig(
```

Comment:
`setCosmosE2EOperationRetryPolicyConfig` to `setCosmosEndToEndOperationRetryPolicyConfig`